WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -230,17 +231,15 @@ public void onCreate() {
         // This call needs be made before accessing any methods in android.webkit package
         setWebViewDataDirectorySuffixOnAndroidP();
 
-        if (CrashLoggingUtils.shouldEnableCrashLogging(getContext())) {
-            CrashLoggingUtils.startCrashLogging(getContext());
-        }
-
         initWellSql();
 
         // Init Dagger
         initDaggerComponent();
         component().inject(this);
         mDispatcher.register(this);
 
+        mCrashLogging.start(getContext());
```

Comment:
By starting this directly without checking for `shouldEnable` first, we're initializing Sentry in debug builds so that we can cause breaking changes to it – if we only ever run it in production, it's harder to catch problems IMHO

Reply:
By starting Sentry directly, I think we made `Sentry` credentials a requirement for production builds. It looks like it still works on `debug` variants without correct credentials, but `release` variants are [throwing an error](https://github.com/wordpress-mobile/WordPress-Android/issues/12120#issuecomment-644193663). (I tried passing a few different mocked versions of the credentials to test)

I've mixed feelings about this. I don't think an outside contributor has to be able to build the `release` variants, but if we can make it a weak dependency easily, we should still do it.

What do you think @jkmassel ?